bundles/core/src/main/java/com/adobe/cq/forms/core/components/internal/PassThroughServlet.java

```diff
@@ -0,0 +1,63 @@
+/*******************************************************************************
+ * Copyright 2021 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+package com.adobe.cq.forms.core.components.internal;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import javax.servlet.Servlet;
+import javax.servlet.ServletException;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.SlingHttpServletResponse;
+import org.apache.sling.api.servlets.SlingAllMethodsServlet;
+import org.osgi.service.component.annotations.Component;
+
+@Component(
+    service = { Servlet.class },
+    property = {
+        "sling.servlet.paths=/bin/ptp/avoidcors",
```

Comment:
Why is this required, no path based servlets and this is not as per the HTTP API spec too.

Reply:
My Bad. This servlet was not supposed to exist here. Was only experimental for testing file attachment preview title. Will remove it right away.